rs/boundary_node/rate_limits/canister/state.rs

```diff
@@ -167,18 +166,17 @@ impl CanisterApi for CanisterState {
     }
 }
 
-pub fn init_version_and_config(version: Version) {
-    with_canister_state(|state| {
-        let config = StorableConfig {
-            schema_version: 1,
-            active_since: time(),
-            rule_ids: vec![],
-        };
-        assert!(
-            state.add_config(INIT_VERSION, config),
-            "Config for version={version} already exists!"
-        );
-    })
+pub fn init_version_and_config(time: Timestamp, canister_api: impl CanisterApi) {
+    // Initialize config with an empty vector of rules
+    let config = StorableConfig {
+        schema_version: INIT_SCHEMA_VERSION,
+        active_since: time,
+        rule_ids: vec![],
+    };
+    assert!(
```

Comment:
This assert serves a slightly different purpose. In the canister `init()` i check that there is some version existing, so that `post_upgrade` doesn't initialize config once again. Whereas here i check that the operation of adding initial `version=1` has succeeded, otherwise canister shouldn't start.